modules/ipi-install-troubleshooting-misc-issues.adoc

```diff
@@ -77,6 +77,29 @@ $ kubectl -n openshift-network-operator logs -l "name=network-operator"
 +
 On high availability clusters with three or more control plane (master) nodes, the Operator will perform leader election and all other Operators will sleep. For additional details, see https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md[Troubleshooting].
 
+== Addressing the `No disk found with matching rootDeviceHints` error
```

Comment:
```suggestion
== Addressing the "No disk found with matching rootDeviceHints" error message
```
- Per [OCP repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#system-messages), `Always refer to a message with the type of message it is, followed by the word "message". For example, refer to an error message as an "error message", and not simply as an "error".`
- [No back ticks allowed in headings](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references).